packages/cadl-typescript/README.md

```diff
@@ -26,15 +26,15 @@ Make sure `npx cadl compile .` runs correctly.
 
 ## Add cadl-typescript
 
-Make sure the version of [cadl-typescript release](https://www.npmjs.com/package/@azure-tools/cadl-typescript) depends on the same version of "@cadl-lang/compiler" as in your Cadl project.
```

Comment:
The folder name `cadl-typescript` should be `typespe-typescript`. Is it changed this time or next?